src/unit_tests.cc

```diff
@@ -56,10 +56,13 @@ UnitTest test_diff{[]()
 
 UnitTest* UnitTest::list = nullptr;
 
-void UnitTest::run_all_tests()
+DurationMs UnitTest::run_all_tests()
```

Comment:
I dont think this function should be measuring the time it takes, it should be done externally.